src/dev-app/dev-app/dev-app-theme.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Inject, Injectable} from '@angular/core';
+import {OverlayContainer} from '@angular/cdk/overlay';
+import {DOCUMENT} from '@angular/common';
+
+const isDarkThemeKey = 'isDarkTheme';
```

Comment:
Add a description what this is for? Maybe also set the value to something like `ANGULAR_COMPONENTS_IS_DARK_THEME`?